src/Rules/Resources/HowToFix.resx

```diff
@@ -347,8 +347,8 @@ If the element can't be resized, ensure the TransformPattern_CanResize property
  4. Height</value>
   </data>
   <data name="BoundingRectangleNotNull" xml:space="preserve">
-    <value>If the element is offscreen, set its IsOffscreen property to TRUE.
-If the element is onscreen, provide a BoundingRectangle property.</value>
+    <value>If the element is off-screen, set its IsOffscreen property to TRUE.
+If the element is on-screen, provide a BoundingRectangle property.</value>
   </data>
```

Comment:
There should be a corresponding changes in `HowToFix.Designer.cs`. These are automatically kept in sync if edited via the IDE, but require manual synchronization if the files are edited outside the IDE.